src/main/java/com/mitchej123/hodgepodge/mixins/late/thaumcraft/MixinTileWandPedestal.java

```diff
@@ -19,34 +20,80 @@
 import thaumcraft.api.aspects.AspectList;
 import thaumcraft.api.visnet.TileVisNode;
 import thaumcraft.api.visnet.VisNetHandler;
+import thaumcraft.common.items.baubles.ItemAmuletVis;
 import thaumcraft.common.items.wands.ItemWandCasting;
 import thaumcraft.common.tiles.TileWandPedestal;
 
 @Mixin(value = TileWandPedestal.class)
-public class MixinTileWandPedestal extends TileThaumcraft {
+public abstract class MixinTileWandPedestal extends TileThaumcraft implements ISidedInventory {
+
+    private int hodgepodge$ticksSinceLastSync = 0;
+    private boolean hodgepodge$needSync;
```

Comment:
Should these be annotated with `@Unique`, or is the hp prefix enough?

https://github.com/2xsaiko/mixin-cheatsheet/blob/master/unique.md